src/jit/gentree.h

```diff
@@ -940,6 +940,9 @@ struct GenTree
 
 #define GTF_ARRLEN_ARR_IDX 0x80000000 // GT_ARR_LENGTH -- Length which feeds into an array index expression
 
+#define GTF_LIST_AGGREGATE 0x80000000 // GT_LIST -- Indicates that this list should be treated as an
```

Comment:
Similar to my previous comment, I think this should be something like GTF_LIST_MULTIREG_ARG